frame/currency-factory/README.md

```diff
@@ -1,41 +1,45 @@
 # Overview
 
-Currency pallet allows to create new sovereign(local consensus) currencies(assets) from thin air.
+This pallet creates new sovereign(local consensus) currencies(assets) from thin air.
 
-Each currency falls into one of next categories:
+Each currency falls into one of the following categories:
 
 - new currency issued by governance
-- currency produced as consequences of execution of protocol and depends on some base currency(ies), usually named LP
-- currency mapping remote location of asset to local identifiers, foreign currency
+- currency produced as consequences of execution of protocol and depended on some base currency(ies), usually named LP
+- currency mapping the remote location of the asset to local identifiers, foreign currency.
+
+Some well-known identifiers are baked into the runtime during compilation, some are added as part of genesis chain specification and some are created in runtime.
+Compilation time currencies are not registered in this pallet.
   
 ## Basics
 
-Currency must be identified, so factory allows to produce new identifiers.
+Currency must be identified, so this pallet allows the production of new identifiers.
 
-Currency to be useful must have amounts. Amounts must be stored on accounts. Accounts must pay ED to prevent spam.
-So each currency must have ED defined.
+A Currency needs an associated amount, associated amounts must be stored on accounts, and accounts must pay ED to prevent spam.
+So each currency must have [ED](../../docs/0002-rent-deposit.md) defined.
```

Comment:
```suggestion
A Currency needs an associated amount stored on accounts. Said accounts must pay ED(Existential Deposit) to prevent spam.
So each currency must have an [ED](../../docs/0002-rent-deposit.md) defined.
```